assets/reducers/popup.js

```diff
@@ -9,7 +10,8 @@ const defaultState = {
     point: null,
     isFetchingElevation: false,
     elevation: null,
-    values: []
+    values: [],
+    region: ''
```

Comment:
Yep. If ```null``` is a valid value, then it's not required. Remove the required part of the prop and all will be well.